sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -554,5 +595,8 @@ def from_estimator(
             multiclass_colors=multiclass_colors,
             xlabel=xlabel,
             ylabel=ylabel,
+            response_method_used=response_method_used,
+            estimator_name=estimator.__class__.__name__,
```

Comment:
I'd add an `name` parameter to `from_estimator` as well, which defaults to `estimator.__class__.__name__` , as the user may no necessarily always want it to be name the estimator name.